tests/check_prog.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+import argparse
+import pathlib
+import os
+import subprocess
+import logging as log
+
+SEED = 3370029442
+
+FILE_DIR = pathlib.Path(__file__).parent.resolve()
+REFERENCE_DIR = FILE_DIR.joinpath("references/")
+
+
+EXIT_SUCCESS = 0
+EXIT_FAILURE = 1
+
+
+def exec_process(cmd, *args, silent=False, **kwargs):
+    log.debug("Executing %s ", cmd)
```

Comment:
This will require a log. Do we have one configured? Usually this is done at the top of the file. 

log = logging.getLogger(__name__)